charts/app-config/values-integration.yaml

```diff
@@ -1415,8 +1415,7 @@ govukApplications:
       redis:
         enabled: true
         redisUrlOverride:
-          app: &link-checker-redis redis://shared-redis-govuk.eks.integration.govuk-internal.digital
-          workers: *link-checker-redis
+          app: redis://shared-redis-govuk.eks.integration.govuk-internal.digital
```

Comment:
Your comment says "switches the web application in integration away from the shared Redis", but this looks like the `app` still points at shared redis, and it's the `workers` that are being switched over.

Is it the comment or the code that's wrong? Or am I misunderstanding?

Reply:
I pushed the wrong branch 🤦‍♂️.  Have corrected that now.